assign_nex_protocols.go

```diff
@@ -36,9 +36,17 @@ func assignNEXProtocols() {
 	// Friends (3DS) protocol handles
 	friends3DSServer.UpdateProfile(friends_3ds.UpdateProfile)
 	friends3DSServer.UpdateMii(friends_3ds.UpdateMii)
-	friends3DSServer.UpdatePreference(friends_3ds.UpdatePreference)
+	friends3DSServer.UpdatePreference(friends_3ds.UpdatePreferences)
```

Comment:
`UpdatePreference` is the real method name. Why was this renamed to `UpdatePreferences`? Function names should mirror the official method names

Reply:
Not actually sure, don't remember the reason. Its been renamed back.